client/components/material-icon/index.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import svg4everybody from 'svg4everybody';
+import spritePath from 'material-design-icons/svg-sprite/material-icons.svg';
```

Comment:
can this be `@automattic/material-design-icons/svg-sprite/material-icons.svg` instead? would remove the need for the alias